node/lib/util/repo_status.js

```diff
@@ -372,7 +372,8 @@ class Submodule {
      */
     isCommittable () {
         return !(this.isNew() &&
-                 null === this.d_index.sha &&
+                 (null === this.d_index ||
+                  null === this.d_index.sha) &&
```

Comment:
nit: you can use optional chaining: `?.`

Reply:
Not with this version of jshint :(